incubator/hnc/test/conversion/conversion_test.go

```diff
@@ -220,13 +220,18 @@ spec:
 		FieldShouldNotContain(hierCRD, nsB, hierSingleton, ".status.conditions", "code")
 	})
 
-	It("should convert HNCConfig", func() {
+	It("should convert HNCConfig and remove all enforced types from spec and only show them in the status", func() {
 		// Convert
 		setupV1alpha2()
 
-		// Verify default resources in the new version.
+		// Verify new apiverison.
 		FieldShouldContainWithTimeout(configCRD, "", configSingleton, ".apiVersion", "v1alpha2", crdConversionTime)
-		FieldShouldContainMultiple(configCRD, "", configSingleton, ".spec.resources", []string{"roles", "rolebindings"})
+		// Verify the enforced types are gone in the new spec.
+		FieldShouldNotContain(configCRD, "", configSingleton, ".spec.resources", "roles")
+		FieldShouldNotContain(configCRD, "", configSingleton, ".spec.resources", "rolebindings")
```

Comment:
Should we just verify that `.spec.resources` is empty, if this is really the default?